cmd/apps/crossplane_app.go

```diff
@@ -0,0 +1,148 @@
+package apps
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path"
+	"strings"
+
+	"github.com/alexellis/k3sup/pkg"
+	"github.com/alexellis/k3sup/pkg/config"
+	"github.com/alexellis/k3sup/pkg/env"
+	"github.com/alexellis/k3sup/pkg/helm"
+	"github.com/spf13/cobra"
+)
+
+func MakeInstallCrossplane() *cobra.Command {
+	var crossplane = &cobra.Command{
+		Use:          "crossplane",
+		Short:        "Install crossplane",
+		Long:         `Install crossplane`,
+		Example:      `  k3sup app install crossplane`,
+		SilenceUsage: true,
+	}
+
+	crossplane.Flags().StringP("namespace", "n", "crossplane-system", "The namespace used for installation")
```

Comment:
Any other flags we may want to override in the chart?

Reply:
I don't think so for the time-being.